### PR TITLE
Nosqlbench 104

### PR DESCRIPTION
**Pull request by jshook**

### CODE REVIEW

File: docsys/src/test/java/io/nosqlbench/docsys/core/PathWalkerTest.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package io.nosqlbench.docsys.core;
+
+import static org.junit.Assert.*;
+
+public class PathWalkerTest {
+
```
Comment (phact) [inline docsys/src/test/java/io/nosqlbench/docsys/core/PathWalkerTest.java]: kill this?

File: engine-api/src/test/java/io/nosqlbench/engine/api/util/NBPathWalkerTest.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class NBPathWalkerTest {
+
+//    @Test
+//    public void testBasicPathmatching() {
+//        List<Path> found = NBPathWalker.findEndMatching(
+//            Path.of("testdocs"),
```
Comment (phact) [inline engine-api/src/test/java/io/nosqlbench/engine/api/util/NBPathWalkerTest.java]: nuke this too?

**Comment by phact** [comment]
LGTM +1

File: docsys/src/test/java/io/nosqlbench/docsys/core/PathWalkerTest.java (hunk 1 above)
Comment (jshook) [reply docsys/src/test/java/io/nosqlbench/docsys/core/PathWalkerTest.java]: will do

File: engine-api/src/test/java/io/nosqlbench/engine/api/util/NBPathWalkerTest.java (hunk 2 above)
Comment (jshook) [reply engine-api/src/test/java/io/nosqlbench/engine/api/util/NBPathWalkerTest.java]: will do in next push as well

**Pull request merged**